.readthedocs.yml

```diff
@@ -1,5 +1,13 @@
-# .readthedocs.yml
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 
-# Don't build any extra formats
-# https://docs.readthedocs.io/en/latest/yaml-config.html#formats
-formats: []
+version: 2
+
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.7"
```

Comment:
This now matches what we actually run to test that we can generate docs in the CI workflow.